### ISSUE TITLE
Questions about the test

### ISSUE THREAD

**Issue by Git-dreamer** [ranrhuang/SPFSplat#17]
Hello! Thank you so much for open-sourcing such an amazing work! However, I’ve encountered some issues and hope you can help me out.

When I test using your pretrained weights `re10k.ckpt` and run the following command:

```bash
python -m src.main +experiment=spfsplat/re10k mode=test wandb.name=re10k \
    dataset/view_sampler@dataset.re10k.view_sampler=evaluation \
    dataset.re10k.view_sampler.index_path=assets/evaluation_index_re10k.json \
    checkpointing.load=./pretrained_weights/re10k.ckpt \
    test.save_image=true test.align_pose=false
```

I get the results from your paper. However, when I modify the testing code by changing:

[pred_extrinsics_cwt[:, v_cxt:],](https://github.com/ranrhuang/SPFSplat/blob/master/src/model/model_wrapper.py#L361)

to:
```bash
batch["target"]["extrinsics"][:, target_view:target_view + 1],
```

which uses the ground truth pose for rendering, the obtained metrics are significantly lower than those obtained before the modification. This seems unreasonable. Do you know what might be causing this issue? Thank you for your time.

┏━━━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━━━━┓
┃        Test metric        ┃       DataLoader 0        ┃
┡━━━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━━━━┩
│          cxt_e_R          │   0.010182916186749935    │
│          cxt_e_t          │   9.061820310307667e-05   │
│           lpips           │    0.21009837090969086    │
│           psnr            │    20.747037887573242     │
│           ssim            │      0.6728555560112      │
│          tgt_e_R          │   0.010120667517185211    │
│          tgt_e_t          │    1.2291628122329712     │
└───────────────────────────┴───────────────────────────┘

**Comment by ranrhuang** [comment]
Hi, thanks for your interest. This is a good question.

pred_extrinsics_cwt[:, v_cxt:] corresponds to the predicted target extrinsics. When you replace it with batch["target"]["extrinsics"][:, target_view:target_view + 1], you are instead using the ground-truth extrinsics.

However, since the model is trained without explicit pose supervision, the predicted poses are only learned to be internally consistent with the reconstructed Gaussians. As a result, they are aligned within the model’s own coordinate frame, which can differ from the ground-truth coordinate system. This mismatch leads to the observed performance drop when substituting ground-truth poses directly.

For this reason, current pose-free methods typically either evaluate using the predicted poses, or apply pose alignment at evaluation time to match the ground-truth frame.

**Comment by Git-dreamer** [comment]
Thank you for your help.
